src/states/game.js

```diff
@@ -24,9 +24,21 @@ Game.prototype =
     {
         ship_mass: 1,
         blackHole_mass: 500000,
-        camera_cutoff_x: 500
+        camera_cutoff_x: 500,
+        start_position: {
+            x: 100,
+            y: 300
+        },
+        score_constant: 0.05
     },
 
+    preload : function()
+    {
+        this.load.bitmapFont('RobotoMono-Regular',
```

Comment:
preload is called when we switch to the 'game' state. It'll be called every time we switch. We don't have to load the font each time we switch to the game state, we only have to load it once (though you would hope that phaser caches which ones it's loaded and which one's it hasn't, but you never know). We should do this load in the 'preload' state's preload function. Similar to how images work (registering them with the preload state in onload).

It would also be good to pull ```RobotoMono-Regular``` into a variable so we can reuse it when we create the text further down:

    this.scoreText = bitmapText( centerX, 45, this.font_name, ... )